changelog/_unreleased/2021-04-15-ignored_url_parameters for cache.md

```diff
@@ -0,0 +1,46 @@
+---
+title:          Ignore some HTTP Parameters from Cache Key generation  
+author:         Björn Herzke  
+issue:          NEXT-14806  
+author_github:  @wrongspot  
+___
+# Storefront
+* Added new Class `Shopware\Storefront\Framework\Cache\CacheKeyProvider`
+* Added configuration `httpcache.ignored_url_parameters` in `storefront.yaml` to blacklist parameters from Cache Key generation
+
+### Usage of Parameter Blacklist
+There are certain use cases where a GET parameter has a new value for every request, generating a new entry in the HTTP cache every time. An example would be the Google Adwords ClickId parameter `gclid` which contains a new id for every click that was generated by Google Adwords. This leads to a bad performance for the visitor since existing caches aren't being used. This allows the caching system to be more efficient.
+
+Storefront Configuration provides a list of known parameters that fall in this category. You can overwrite or extend this list in your [bundle configuration](https://developer.shopware.com/docs/v/v6.4.0/guides/hosting/infrastructure/filesystem#configuration).   
+```
+storefront:
+    httpcache:
+        ignored_url_parameters:
+            - 'pk_campaign'     # Piwik
```

Comment:
I would comment with the new name

Reply:
I'm not with you 🤔 There may be people using the "old" Piwik